shared/sliceutil/slice.go

```diff
@@ -83,6 +83,22 @@ func UnionUint64(s ...[]uint64) []uint64 {
 	return set
 }
 
+// CleanUint64 removes duplicate indices from
+// the slice.
+func CleanUint64(a []uint64) []uint64 {
```

Comment:
Clean as a term is ambiguous in this case. How about `SetUint64`?

Reply:
`SetUint64` also doesn't describe what this method actually does, its too generic. Any other names you think would work ? I couldn't find a good name to describe it

Reply:
Set means deduplicates (ie pyhton, java)  in most context.  What meaning of Set are you referring to that doesn't describe?

Reply:
`Set` meaning setters/getters . `SetUint64` reads as though your are setting the value to these indices, which wouldn't be wholly correct.  Reading your link you are referring to a mathematical set, which would make more sense.

Reply:
ok changed, done

Reply:
I see. I was more referring to spec uses Set for duplicates: https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#get_attesting_indices

But I do see the confusion :)